qemu/tests/single_driver_install.py

```diff
@@ -133,7 +133,9 @@ def run(test, params, env):
         _add_cert(session, cert, store)
 
     error_context.context("Installing target driver", logging.info)
-    inst_cmd = "%s updateni %s %s" % (devcon_path, inf_path, device_hwid)
+    inst_cmd = ["%s updateni %s %s" % (devcon_path, inf_path, hwid)
+                for hwid in device_hwid.split()]
+    inst_cmd = " || ".join(inst_cmd)
     status, output = session.cmd_status_output(inst_cmd, inst_timeout)
```

Comment:
Do we need to send the inst_cmd separately via cmd_status_output?
So that we can know exactly which inst_cmd is the cause.

Reply:
ok, makes sense to me, will update.